pkg/crinit/aggregated/aggregated.go

```diff
@@ -186,18 +192,18 @@ func Run(opts *aggregatedClusterRegistryOptions, cmdOut io.Writer,
 		return err
 	}
 
-	err = opts.CreateAPIServer(cmdOut, hostClientset, false, false, ips, pvc)
+	sa, err := createRBACObjects(cmdOut, hostClientset, opts)
```

Comment:
Why is this an odd man out? Why does this take opts as an argument instead of taking it as a receiver like other method calls around?

Reply:
Hmm. I may have just missed this in review. @font?

I think we intend to refactor this anyway, so that opts is not the receiver, so I'm not so concerned about having this be inconsistent for a bit.

Reply:
We can fix this in a follow-up.  Remember, this isn't even alpha yet :)

Reply:
No particular reason other than this function is in this package whereas the others are in the `util` package which contains the defined `SubcommandOptions` structure. This `opts` gets passed as the `aggregatedClusterRegistryOptions` type in this package whereas the other methods in `util` are invoked as `opts.SubcommandOptions` (the embedded type) receivers.

Also as @perotinus states, this is going to be refactored further as there is an open PR #127 for it that may already take care of it (haven't reviewed it yet).

Reply:
@perotinus @pmorie @font fair enough. Good to know that you are all aware of it and there is an open PR to refactor it.